### PR TITLE
virttest/qemu_vm.py: Add new func get_qemu_threads()

### PR DESCRIPTION
**Pull request by spcui**
Add new function to get the qemu threads.

Signed-off-by: Shuping Cui <scui@redhat.com>

ID: 1297288

### CODE REVIEW

File: virttest/qemu_vm.py [hunk 1]
```diff
@@ -3045,6 +3045,20 @@ def get_pid(self):
         except (TypeError, IndexError, ValueError):
             return None
 
+    def get_qemu_threads(self):
+        """
```
Comment (xutian) [inline virttest/qemu_vm.py]: you can list tasks launched by qemu by command
"ls /proc/%d/task"  % self.get_pid()

**Comment by spcui** [comment]
@xutian updated, please help review again?

File: virttest/qemu_vm.py [hunk 2]
```diff
@@ -3045,6 +3045,19 @@ def get_pid(self):
...
+        Return the list of qemu SPIDs
+
+        :return: the list of qemu SPIDs
+        """
+        try:
+            cmd = "ls /proc/%d/task" % self.get_pid()
+            threads_list = commands.getoutput(cmd).split()
+            return threads_list
+        except (TypeError, IndexError, ValueError):
```
Comment (xutian) [inline virttest/qemu_vm.py]: useless catch!!!

File: virttest/qemu_vm.py [hunk 3]
```diff
@@ -3045,6 +3045,16 @@ def get_pid(self):
...
             return None
 
+    def get_qemu_threads(self):
+        """
+        Return the list of qemu SPIDs
+
+        :return: the list of qemu SPIDs
+        """
+        cmd = "ls /proc/%d/task" % self.get_pid()
+        threads_list = commands.getoutput(cmd).split()
```
Comment (xutian) [inline virttest/qemu_vm.py]: if status not equal 0, output are error message,  output looks like:  
>>> z = commands.getoutput("kk")
>>> z
'sh: kk: command not found'
so, I think you should check status of command before return it.

File: virttest/qemu_vm.py [hunk 4]
```diff
@@ -3656,11 +3668,18 @@ def migrate(self, timeout=virt_vm.BaseVM.MIGRATE_TIMEOUT, protocol="tcp",
...
 
+            threads_list1 = self.get_qemu_threads()
+
             logging.info("Migrating to %s", uri)
             self.monitor.migrate(uri)
             if not_wait_for_migration:
                 return clone
 
+            threads_list2 = self.get_qemu_threads()
+            if threads_list1 == threads_list2:
```
Comment (xutian) [inline virttest/qemu_vm.py]: It should be  if len(thread_list2) <= len(thread_list1) test failed.

**Comment by xutian** [comment]
looks ok to me. Merged. thanks @spcui

**Pull request merged**